responders/src/UI/embc-responder/src/app/shared/pipes/maskEvacuatedAddress.pipe.ts

```diff
@@ -0,0 +1,54 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { AddressModel } from 'src/app/core/models/address.model';
+import { LocationsService } from 'src/app/core/services/locations.service';
+import * as _ from 'lodash';
+
+@Pipe({ name: 'maskEvacuatedaddress' })
+export class MaskEvacuatedAddressPipe implements PipeTransform {
+  constructor(
+    private locationService: LocationsService,
+    private sanitizer: DomSanitizer
+  ) {}
+
+  /**
+   * Converts Address object into sanitized 2-line format for display on pages.
+   * Due to HTML sanitization, cannot be {{ interpolated }}, must use [innerHTML]
+   *
+   * @param address Address object to display on page
+   * @returns Sanitized Two-line SafeHTML string, <br> as only included HTML
+   */
+  transform(address: AddressModel): SafeHtml {
+    if (address !== null && address !== undefined) {
+      const communities = this.locationService.getCommunityList();
+
+      let line1 = address.addressLine1;
+      let line2 = '';
+
+      if (address.addressLine2?.length > 0)
+        line1 += ', ' + address.addressLine2;
+
+      // If community code is found, use that. Otherwise, use city
+      const communityName =
```

Comment:
This mapping should not be part of the map but the service when profile is getting created